src/files/coverageparser.ts

```diff
@@ -213,17 +216,18 @@ export class CoverageParser {
                     return resolve(sections);
                 });
             } catch (error) {
-                this.handleError("lcov-parse", error);
+                this.handleError("lcov-parse", error, filename);
```

Comment:
instead of adding filename to the `handleError` signature can we instead construct a new variable that appends the filename to the error message?
```
} catch (error) {
  error.message = `filename: ${filename} ${error.message}`
  this.handleError("lcov-parse", error);
}
```